spa/src/components/account/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,100 @@
+import { useLocation } from 'react-router-dom';
+import { useState, useReducer, useMemo } from 'react';
+
+// AJAX
+import axios from 'ajax/axios';
+import { RESET_PASSWORD_ENDPOINT } from 'ajax/endpoints';
+
+// State management
+import fetchReducer, { initialState, FETCH_START, FETCH_SUCCESS, FETCH_FAILURE } from 'state/fetch-reducer';
+
+import * as S from '../Account.styled';
+
+import ResetPasswordForm from './ResetPasswordForm';
+import Logobar from 'components/account/common/logobar/Logobar';
+import Leftbar from 'components/account/common/leftbar/Leftbar';
+
+import { SIGN_IN } from 'routes';
+import YellowSVG from 'assets/images/account/yellow-bar.svg';
+
+// Analytics
+import { useConfigureAnalytics } from 'components/analytics';
+
+function FetchQueryParams() {
+  const { search } = useLocation();
+  return useMemo(() => new URLSearchParams(search), [search]);
+}
+
+function ResetPassword() {
+  useConfigureAnalytics();
+
+  let qParams = FetchQueryParams();
+  const token = qParams.get('token');
```

Comment:
If `FetchQueryParams` and `qParams` are being used only here, I think we don't need a separate function/constant for this:

```suggestion
function ResetPassword() {
  useConfigureAnalytics();
  const { search } = useLocation();
  const token = useMemo(() => new URLSearchParams(search).get('token'), [search]);
```

Reply:
done